app/src/ui/lib/git-perf.ts

```diff
@@ -18,7 +18,6 @@ export async function measure<T>(
 ): Promise<T> {
   const id = ++markID
 
-  log.debug(`Executing ${cmd}`)
```

Comment:
does this mean this will no longer be in production logs? i've found knowing about all git operations helpful for debugging production stuff in some cases. what's the aim of removing this?

Reply:
> does this mean this will no longer be in production logs?

`log.debug` messages would only appear in the Dev Tools console for production builds. I wasn't sure if anyone was getting any value out of that there, and it duplicated information as a result of the "always log timings when in development mode" change so I took it out.

Happy to add this back, but rather than adding this back immediately I'll ask: "what information about Git operations would you like to know when using a production build of the app?"

Reply:
ah ok! i think i misremembered where i was looking at logs so i think i'm 👍 with this change. thanks for the extra info.